### PR TITLE
Clearer doc for the config: URL

### PR DESCRIPTION
**Pull request by sbrunner**
related to #1591

### CODE REVIEW

File: doc/administrator/administrate.rst [hunk 1]
```diff
@@ -133,11 +133,17 @@ URL
...
+  * ``static://prj:img/icon.png`` will get the URL of the ``img`` static route of ``prj``.
+
+* ``config``: to get the server name from the URL, with the config from the ``vars`` file:
+
+  .. code:: yaml
+
+     servers:
+        my_server: http://example.com/test
+
+  ``config://my_server/icon.png``, will be transformed in
```
Comment (asaunier) [inline doc/administrator/administrate.rst]: transformed into

File: doc/administrator/administrate.rst [hunk 2]
```diff
@@ -133,11 +133,17 @@ URL
...
-  * 'static:///icon.png' will get the URL of the 'static' static route of the project.
-  * 'static://c2cgeoportal/icon.png' will get the URL of the 'static' static route of 'c2cgeoportal'.
-  * 'static://prj:img/icon.png' will get the URL of the 'img' static route of 'prj'.
-
-* ``config``: to get the server name from the url,
-  if we use 'config://my_server/icon.png', with the config:
-  `servers: { "my_server": "http://example.com/test" }`
-  we will get the URL: 'http://example.com/test/icon.png'
+
+  * ``static:///icon.png`` will get the URL of the ``static`` static route of the project.
```
Comment (asaunier) [inline doc/administrator/administrate.rst]: The triple / is on purpose I guess?

File: doc/administrator/administrate.rst [hunk 3]
```diff
@@ -133,11 +133,17 @@ URL
...
+  * ``static://c2cgeoportal/icon.png`` will get the URL of the ``static`` static route of ``c2cgeoportal``.
+  * ``static://prj:img/icon.png`` will get the URL of the ``img`` static route of ``prj``.
+
+* ``config``: to get the server name from the URL, with the config from the ``vars`` file:
+
+  .. code:: yaml
+
+     servers:
```
Comment (rbovard) [inline doc/administrator/administrate.rst]: Extra space before `servers:`

File: doc/administrator/administrate.rst (hunk 2 above)
Comment (sbrunner) [inline doc/administrator/administrate.rst]: The triple `/` is wanted, `static://c2cgeoportal/icon.png` is different than `static:///c2cgeoportal/icon.png`, the first point the file `/icon.png` in c2cgeoportal, the second `/c2cgeoportal/icon.png` in the project.

File: doc/administrator/administrate.rst (hunk 3 above)
Comment (sbrunner) [inline doc/administrator/administrate.rst]: No it's coorect, it should be align with the `code::` :-)

Comment (rbovard) [inline doc/administrator/administrate.rst]: Ok...

**Pull request merged**